fcrepo-connector-file/src/main/java/org/fcrepo/connector/file/FedoraFileSystemConnector.java

```diff
@@ -324,12 +322,10 @@ protected void touchParent( final String id ) {
 
     /* Overriding so unit test can mock. */
     @Override
-    @VisibleForTesting
```

Comment:
If they are helpful, that's different, but may I ask _how_ are they helpful? My understanding is that this annotation will relax the visibility of private methods during testing. All of our uses of it, though, are as decorators on protected and public functions.